tests/test_request_cache.py

```diff
@@ -57,6 +57,39 @@ def foo(a, b, c, x=None, y=None, z=None):
     )
 
 
+@pytest.mark.parametrize('cache_set_call, expected_redis_call', (
+    (0, 0),
+    (1, 1),
+    (1.111, 1),
```

Comment:
Optional, could also add in a string case to this like `'2000'` becomes `2000`

Reply:
```suggestion
    (1.111, 1),
    ('2000', 2_000),    
```